components/HighlightedLinks/src/stories/bem.stories.mdx

```diff
@@ -30,146 +30,138 @@ import readme from "../../README.md";
-                  viewBox="0 0 24 24"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
-                  class="denhaag-icon"
-                  focusable="false"
-                  aria-hidden="true"
-                  shape-rendering="auto"
-                >
-                  <path
-                    d="M12.293 5.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L16.586 13H5a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z"
-                    fill="currentColor"
-                  ></path>
-                </svg>
-              </span>
-              <span>Inburgeren en naturaliseren</span>
-            </a>
-          </li>
-          <li class="denhaag-link-group__navigation-list-item">
-            <a
-              href="https://nl-design-system.github.io/denhaag/"
-              class="denhaag-link denhaag-link--with-icon denhaag-link--with-icon-start"
-            >
-              <span class="denhaag-link__icon">
-                <svg
-                  width="1em"
-                  height="1em"
-                  viewBox="0 0 24 24"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
-                  class="denhaag-icon"
-                  focusable="false"
-                  aria-hidden="true"
-                  shape-rendering="auto"
-                >
-                  <path
-                    d="M12.293 5.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L16.586 13H5a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z"
-                    fill="currentColor"
-                  ></path>
-                </svg>
-              </span>
-              <span>Akten</span>
-            </a>
-          </li>
-          <li class="denhaag-link-group__navigation-list-item">
-            <a
-              href="https://nl-design-system.github.io/denhaag/"
-              class="denhaag-link denhaag-link--with-icon denhaag-link--with-icon-start"
-            >
-              <span class="denhaag-link__icon">
-                <svg
-                  width="1em"
-                  height="1em"
-                  viewBox="0 0 24 24"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
-                  class="denhaag-icon"
-                  focusable="false"
-                  aria-hidden="true"
-                  shape-rendering="auto"
-                >
-                  <path
-                    d="M12.293 5.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L16.586 13H5a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z"
-                    fill="currentColor"
-                  ></path>
-                </svg>
-              </span>
-              <span>Verklaringen</span>
-            </a>
-          </li>
-        </ul>
-      </nav>
+      <ul class="utrecht-link-list utrecht-link-list--html-ul denhaag-link-group__navigation-list denhaag-highlighted-links__navigation-list">
```

Comment:
Is it possible to reduce the css-selectors? Something like `denhaag-highlighted-links__list` instead of `denhaag-highlighted-links__navigation-list` other items will have `denhaag-link-group__list-item`.

This will reduce the CSS quite a bit.